<?xml version="1.0" encoding="UTF-8" ?>
<testsuite errors="0" failures="1" hostname="EX064" name="testgrails.ConfigurationTests" tests="1" time="1.829" timestamp="2013-02-28T15:39:38">
  <properties />
  <testcase classname="testgrails.ConfigurationTests" name="testSomething" time="0.969">
    <failure message="junit.framework.AssertionFailedError: Implement me" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: Implement me
	at testgrails.ConfigurationTests.testSomething(ConfigurationTests.groovy:15)
</failure>
  </testcase>
  <system-out><![CDATA[--Output from testSomething--
]]></system-out>
  <system-err><![CDATA[--Output from testSomething--
]]></system-err>
</testsuite>
